app/src/main/java/com/codesoom/assignment/service/TaskService.java

```diff
@@ -0,0 +1,46 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.model.Task;
+import com.codesoom.assignment.repository.TaskRepository;
+import com.codesoom.assignment.repository.TaskRepositoryImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TaskService {
+
+    private final TaskRepository taskRepository;
+
+    @Autowired
+    public TaskService(TaskRepositoryImpl taskRepository) {
```

Comment:
```suggestion
    public TaskService(TaskRepository taskRepository) {
```

구현체가 들어오면 DI의 의미가 퇴색됩니다!